### PR TITLE
Added flag to prevent multiple triggers of getGroupingOwnersOnClick

### PR DESCRIPTION
**Pull request by jarellb**
# Ticket Link

[GROUPINGS-1765](https://uhawaii.atlassian.net/jira/software/c/projects/GROUPINGS/boards/18?assignee=712020%3Ad0e5c67f-f1f1-44a0-b5c9-282af528541e&selectedIssue=GROUPINGS-1765)

# List of squashed commits

1. Fixed codacy issue and shifted $scope.loadingOwner variables
2. Corrected coding standard issue

# Test Checklist

- [x] Exhibits Clear Code Structure:
- [x] Project Unit Tests Passed:
- [x] Project Jasmine Tests Passed:
- [x] Executes Expected Functionality:
- [x] Tested For Incorrect/Unexpected Inputs:

### CODE REVIEW

**Comment by JorWo** [comment]
Just a very minor nitpick, I would want the `$scope.loadingOwner` variable inside `$scope.handleGroupingOwnersOnError` and `$scope.handleGroupingOwnersOnSuccess` to be placed right under `$scope.loading`. 

There is also a Codacy issue that you can find in the list of checks at the bottom of this PR. I personally like that coding style you used for early returns in a function but we will need to follow the code style for this project in particular.

File: src/main/resources/static/javascript/mainApp/admin.controller.js [hunk 1]
```diff
@@ -469,6 +472,8 @@
          * @param groupingPath - The path of the grouping to display owners
          */
         $scope.getGroupingOwnersOnClick = (groupingPath) => {
+            if ($scope.loadingOwners === true) {return;} // Prevent multiple triggers while loading modal
```
Comment (JorWo) [inline src/main/resources/static/javascript/mainApp/admin.controller.js:475]: Sorry, one more thing if you could expand the brackets like this:
```
// Prevent multiple triggers while loading modal
if ($scope.loadingOwners === true) {
    return;
}
```
 Turns out Codacy gave a terrible suggestion lol

Comment (jarellb) [reply src/main/resources/static/javascript/mainApp/admin.controller.js:475]: Okay got it, thank you

**Review by JorWo — APPROVED** [review]
Great work!

**Pull request merged**